laravel-app/database/migrations/2023_09_19_021715_create_users_table.php

```diff
@@ -18,6 +18,7 @@ public function up(): void
             $table->boolean('is_verified')->default(false);
             $table->string('password');
             $table->rememberToken();
+            $table->timestamps();
```

Comment:
why you updated this??? please dont update the previous migration files..

Reply:
removed changes on users migration file, added new file to add timestamps column on existing users table